presto-main/src/test/java/io/prestosql/spiller/TestFileSingleStreamSpillerFactory.java

```diff
@@ -75,20 +76,30 @@ public void tearDown()
         closer.close();
     }
 
-    @Test
-    public void testDistributesSpillOverPaths()
-            throws Exception
+    private FileSingleStreamSpillerFactory spillerFactoryFactory(List<Path> paths)
     {
-        List<Type> types = ImmutableList.of(BIGINT);
-        List<Path> spillPaths = ImmutableList.of(spillPath1.toPath(), spillPath2.toPath());
-        FileSingleStreamSpillerFactory spillerFactory = new FileSingleStreamSpillerFactory(
+        return spillerFactoryFactory (paths, 1.0);
+    }
+
+    private FileSingleStreamSpillerFactory spillerFactoryFactory(List<Path> paths, Double maxUsedSpaceThreshold)
```

Comment:
please extract this change to separate commit, before your actual change. Because it is just a refactoring. Also please put this method somewhere at the bottom, it is not just much important as it is only utility method.